code/game/machinery/suit_storage_unit.dm

```diff
@@ -153,6 +153,7 @@
 
 /obj/machinery/suit_storage_unit/Initialize(mapload)
 	. = ..()
+	obj_flags |= USES_TGUI
```

Comment:
Add this in the definition for `/obj/machinery/suit_storage_unit` by setting obj_flags = (whatever /obj/machinery has) | USES_TGUI

Reply:
pushed now, thanks for the feedback!